src/glmfit.jl

```diff
@@ -421,33 +421,47 @@ function StatsBase.fit!(m::AbstractGLM,
     end
 end
 
+const FIT_GLM_DOC = """
+    In the first method, `formula` must be a `Formula` object and `data` a table
+    (in the Tables.jl definition, e.g. a data frame).
```

Comment:
```suggestion
    In the first method, `formula` must be a [StatsModels.jl `Formula` object](https://juliastats.org/StatsModels.jl/stable/formula/)
    and `data` a table (in the Tables.jl definition, e.g. a data frame).
```

Might not be clear to users what the formula object means... (also elsewhere with the same language)